pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/buffer/Bytes.java

```diff
@@ -556,4 +556,108 @@ private static Comparator<Bytes> valueSorter(@NonNull final Comparator<Byte> byt
             return ((len > 0) ? 1 : -1);
         };
     }
+
+    /** {@inheritDoc} */
+    public int readVarInt(int[] pos, final boolean zigZag) {
```

Comment:
These two methods need to be renamed as @override of interface as explained in PR #57

Reply:
Same as above.